makefile

```diff
@@ -0,0 +1,8 @@
+.PHONY: .venv
+
+run-test:
```

Comment:
okay this is nice start for preparening makefile, but you use wrong name, it has to be in capital for the first letter, so instead makefile rename it to Makefile

Reply:
done